bash/scripts/locate_release_candidate.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+main() {
+  envPropsFilepath="${BATS_TEST_DIRNAME}/tmp/env.properties"
+  if [ -n "${JENKINS_HOME}" ]; then
+    envPropsFilepath="${WORKSPACE}/env.properties"
+  fi
+
+  tag=''
+  if [ -n "${TAG}" ]; then
+    echo "TAG set to '${TAG}', attempting release of this tag..."
+    tag="${TAG}"
+  else
+    if [ -n "${GIT_BRANCH}" ]; then
+      echo "GIT_BRANCH set to '${GIT_BRANCH}', attempting release of this tag..."
+      tag="${GIT_BRANCH#origin/tags/}"
+    else
+      echo "GIT_BRANCH not set, cannot determine tag to release; exiting."
+      exit 1
+    fi
+
+    # make sure tag matches latest; else, exit
```

Comment:
Would there ever be a case where we'd want to release an older tag like a patch release for an older release?